splash/qtutils.py

```diff
@@ -166,7 +166,7 @@ def qt2py(obj, max_depth=100):
 
     # print(obj, obj.__class__)
 
-    if isinstance(obj, QString):
+    if isinstance(obj, (QString,QByteArray)):
         return unicode(obj)
```

Comment:
I think that supporting unicode but returning null as '' is a reasonable tradeoff. And anyways, this is how it already works for splash:jsfunc, splash:runjs and splash:evaljs :) See http://splash.readthedocs.org/en/master/scripting-ref.html#js-lua-conversion-rules.